prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1361,45 +1306,39 @@ int rxr_ep_init(struct rxr_ep *ep)
 		pkt_pool_flags = OFI_BUFPOOL_HUGEPAGES;
 	}
 
-	ret = rxr_create_pkt_pool(ep, entry_sz, rxr_get_tx_pool_chunk_cnt(ep),
-				  pkt_pool_flags,
-				  &ep->efa_tx_pkt_pool);
+	ret = rxr_pkt_pool_create(ep, entry_sz, rxr_get_tx_pool_chunk_cnt(ep),
+				  rxr_get_tx_pool_chunk_cnt(ep), pkt_pool_flags, true, &ep->efa_tx_pkt_pool);
```

Comment:
To help reader understand, add a comment around  important arguments.

Something like

```
 rxr_pkt_pool_create(ep, entry_sz, rxr_get_tx_pool_chunk_cnt(ep),
				  rxr_get_tx_pool_chunk_cnt(ep) /* max_count */,
                                  pkt_pool_flags,
                                 true /* register_memory */,
                                 &ep->efa_tx_pkt_pool);
```